src/pudl/cli.py

```diff
@@ -123,18 +123,18 @@ def main():
 
     etl_settings = EtlSettings.from_yaml(args.settings_file)
 
-    if (not os.getenv("PUDL_OUT")) or (not os.getenv("PUDL_CACHE")):
+    if (not os.getenv("PUDL_OUT")) or (not os.getenv("PUDL_INPUT")):
```

Comment:
I noticed this elsewhere, but I think we need to check for `PUDL_OUTPUT` instead of `PUDL_OUT`

Reply:
Good catch! Also we need to set these properly in conftest, at least until we completely rip out the old pudl_settings.